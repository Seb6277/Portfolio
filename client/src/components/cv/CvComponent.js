import React from 'react'

class CvComponent extends React.Component{
  render() {
    return(
      <div className="cv_frame">
        <h2>CV</h2>
      </div>
    )
  }
}

export default CvComponent