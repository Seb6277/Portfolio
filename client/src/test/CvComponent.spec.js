import React from 'react'
import {shallow} from 'enzyme'
import renderer from 'react-test-renderer'

import CvComponent from '../components/cv/CvComponent'

describe('<CvComponent/>', () => {
  it('should render without crash', () => {
    const wrapper = shallow(<CvComponent/>)
    expect(wrapper)
  });

  it('should match snapshot', () => {
    const component = renderer.create(<CvComponent/>)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
    tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  });
})