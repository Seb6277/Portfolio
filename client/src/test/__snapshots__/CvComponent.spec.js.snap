// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CvComponent/> should match snapshot 1`] = `
<div
  className="cv_frame col-md-8 offset-md-2"
>
  <h1>
    CV
  </h1>
  <p>
    Vous pouvez télécharger mon CV par le liens ci-dessous:
  </p>
  <a
    href="cv.pdf"
  >
    <img
      alt="TODO"
      src="document-download-solid.png"
    />
  </a>
</div>
`;

exports[`<CvComponent/> should match snapshot 2`] = `
<div
  className="cv_frame col-md-8 offset-md-2"
>
  <h1>
    CV
  </h1>
  <p>
    Vous pouvez télécharger mon CV par le liens ci-dessous:
  </p>
  <a
    href="cv.pdf"
  >
    <img
      alt="TODO"
      src="document-download-solid.png"
    />
  </a>
</div>
`;
